src/components/InputDropdown/InputDropdown.styles.js

```diff
@@ -1,13 +1,19 @@
 import { css } from '@emotion/core';
 import { Borders, Color, selectReset, Radius, Space } from 'lib/theme';
 
+const INPUT_HEIGHT = 65;
```

Comment:
Can we find a place to hold this so it can be shared between both inputs? And any that come in the future.